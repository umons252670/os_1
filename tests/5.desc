concurrent test of trace() and gettracecount()
