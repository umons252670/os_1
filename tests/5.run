cd src; ../run-xv6-command.exp CPUS=2 Makefile.test test_5 | grep XV6_TEST_OUTPUT; cd ..
