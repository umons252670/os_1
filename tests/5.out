XV6_TEST_OUTPUT 0 0 3000 3001
